tests/ellipsoidal_chunk_geometry.cc

```diff
@@ -0,0 +1,98 @@
+#include <aspect/geometry_model/ellipsoidal_chunk.h>
+#include <aspect/geometry_model/initial_topography_model/zero_topography.h>
+
+#include <deal.II/base/exceptions.h>
+
+#include <iostream>
+
+using namespace aspect;
+using namespace dealii;
+
+bool test_point(const GeometryModel::EllipsoidalChunk<3>::EllipsoidalChunkGeometry ellipsoidal_manifold,
+                const Point<3> &test_point)
+{
+  const Point<3> converted_point = ellipsoidal_manifold.pull_back(test_point);
+  const Point<3> twice_converted_point = ellipsoidal_manifold.push_forward(converted_point);
+
+  const Tensor<1,3> difference = twice_converted_point - test_point;
+  const double error = difference.norm() / test_point.norm();
+
+  std::cout << "Point: " << test_point
+            << " becomes: " << converted_point
+            << ". Back transformed: " << twice_converted_point
+            << ". Error: " << error << std::endl;
+
+  return true;
+}
+
+/*
+ * Launch the following function when this plugin is created.
+ */
+int f()
+{
+
+  const unsigned int dim=3;
+
+  InitialTopographyModel::ZeroTopography<dim> topography;
+  GeometryModel::EllipsoidalChunk<dim>::EllipsoidalChunkGeometry ellipsoidal_manifold;
+  ellipsoidal_manifold.initialize(&topography);
+
+  std::vector<Point<2> > corners(2,Point<2>(-15.0,-15.0));
+  corners[1] *= -1.0;
+
+  // Simple sphere
+  ellipsoidal_manifold.set_manifold_parameters(6371000.0,
+                                               0.0,
+                                               6371000.0,
+                                               2890000.0,
+                                               corners);
+
+  std::vector<Point<3> > test_points;
+  test_points.push_back(Point<3> (6371000.0,0,0));
+  test_points.push_back(Point<3> (6171000.0,0,0));
+  test_points.push_back(Point<3> (3000000.0,3000000.0,0));
+  test_points.push_back(Point<3> (3000000.0,3000000.0,3000000.0));
+  test_points.push_back(Point<3> (3000000.0,-3000000.0,-3000000.0));
+  test_points.push_back(Point<3> (25000.0,25000.0,3481000.0));
+  test_points.push_back(Point<3> (25000.0,25000.0,1000000.0));
+  test_points.push_back(Point<3> (25000.0,25000.0,500000.0));
+  test_points.push_back(Point<3> (25000.0,25000.0,50000.0));
+
```

Comment:
Can you for the sake of clarity in the output put a line in there which outputs something like "simple sphere test:". And also do that for the others?